### PR TITLE
tailwind example added

### PR DESCRIPTION
**Pull request by Anishpras**
I have added a TailwindCSS example with configurations.

### CODE REVIEW

File: examples/with-tailwind/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix!
```
Comment (sergiodxa) [inline examples/with-tailwind/README.md:1]: Did you create this copying the template folder or running the Remix CLI?

Comment (Anishpras) [reply examples/with-tailwind/README.md:1]: @sergiodxa I ran Remix CLI for this.

Comment (Anishpras) [reply examples/with-tailwind/README.md:1]: @sergiodxa Is there any issue you want me to resolve?

**Review by kentcdodds — COMMENTED** [review]
Thanks for taking the time to put this together. We're going to go with this PR instead: https://github.com/remix-run/remix/pull/1373

**Pull request closed without merging**